src/security/issues/builder.py

```
@@ -70,10 +81,9 @@ def classify_category(alert: Alert) -> str:
     return alert.metadata.rule_name
 
 
-def build_parent_issue_title(rule_id: str, severity: str = "") -> str:
+def build_parent_issue_title(rule_id: str) -> str:
     """Build the title string for a parent issue."""
-    sev_tag = f"[{severity.upper()}] " if severity else ""
-    return f"{sev_tag}Security Alert – {rule_id}".strip()
+    return f"Security Alert – {rule_id}".strip()
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)